CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- `observer`: Document available observers (#10889)
```

Comment:
I've added the skip changelog tag, we don't generally document doc changes in the changelog 👍🏻

Reply:
Fixed.